### DISCUSSION TITLE
Quartet

### DISCUSSION THREAD

**Discussion by darylhjd** [SethClydesdale/genki-study-resources#278] [Discussions]
Hello,

Just wondering if you have any plans to create something similar for the Quartet series!

**Comment by SethClydesdale** [comment]
Hello and sorry for the late response!

I currently don't have any plans to make a project similar to this for Quartet, though, it's not like I haven't considered it. It's just that I'm more focused on other things at the moment, such as improving my translation skills and studying regional dialects. This may sound selfish, but if I were to take on another project like this, it'd require me to sacrifice a lot of my time again for essentially nothing, whereas with Genki and Tobira there was a lot of new knowledge for me to acquire along the way that helped keep me motivated.

At the very least, there's the Genki and [Tobira resources](https://sethclydesdale.github.io/tobira-study-resources/) that you can supplement your studies with, as they both teach a lot of the same stuff. So I hope that those will still be of help to you and others!

**Comment by SethClydesdale** [comment]
Hello again!

The last time I wrote I was feeling a little burnt out, but having taken a break to reflect on some things, I believe I may begin development on resources for Quartet sometime next year. It ought to be a good chance to review on my part, but also seeing how many people want resources for Quartet makes me want to deliver a similar experience to Genki Study Resources that'll help them with their studies.

**Comment by darylhjd** [comment]
hey!

appreciate all you have done and what you will be doing in the near future :D

please feel free to reach out to the community if you ever need help and do remember to give yourself a well deserved break!
